internal/status/ing_status.go

```diff
@@ -478,6 +478,7 @@ func getIngresses(ingressNSNames []string, bulk bool, retryNum ...int) map[strin
 		// to return all AKO ingestable Ingresses.
 		aviIngClasses := make(map[string]bool)
 		if utils.GetIngressClassEnabled() {
+			//It seems v1bet1/ingressclass deprecated in 1.19+. Should we use networking/v1 ingress class ?
```

Comment:
hey can we remove the comment? updating to networkingv1 will be an epic and will require refactors.